AllReadyApp/Web-App/AllReady.UnitTest/Features/Tasks/AddTaskCommandHandlerAsyncTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Threading.Tasks;
+using AllReady.Features.Tasks;
+using AllReady.Models;
+using Moq;
+using Xunit;
+
+namespace AllReady.UnitTest.Features.Tasks
+{
+    public class AddTaskCommandHandlerAsyncTests
```

Comment:
Just a note on naming tests...not needed at all, but if you use the class name as part of the test intent description, you get `class AddTaskCommandHandlerShould` and then a test name of `InvokeAddTaskHandlerWithCorrectData` and the two read like a sentence in test suites.